README.md

```diff
@@ -27,6 +27,17 @@ contact our [support team](https://support.adyen.com/hc/en-us/requests/new?ticke
 ## Integration
 The plugin integrates card component (Secured Fields) using Adyen Checkout for all card payments.
 
+### Supported payment methods
+ - Credit and debit cards (non 3D secure, 3D secure 1, 3D secure 2 native)
+ - Stored card payment methods (One click payment methods)
+ - iDeal
+ - Klarna Pay Later
+ - Klarna Pay Over Time
+ - Klarna Pay Now
+ - PayPal (redirect)
+ - SEPA Direct Debit
+ - Sofort
+
```

Comment:
I see the Giropay handler, but maybe it isn't ready?

Reply:
indeed It is missing from this list